jupyter-book/preamble.md

```diff
@@ -12,53 +12,57 @@ Interested in contributing? Please have a look at https://github.com/theislab/si
+
+## Who should read this book
+
+This book is designed for a diverse audience with an interest in single-cell data analysis, including **biologists** looking to gain practical data analysis skills, **computer scientists** seeking to explore the biological underpinnings and computational challenges of single-cell data, and **bioinformaticians** aiming to refine their knowledge or stay updated with best practices. Whether you’re a beginner eager to learn the fundamentals or an experienced analyst striving to enhance your expertise and adopt effective workflows, this book provides the support you need to navigate the complexities of single-cell analysis.
 
 ## Structure of the book
 
-Each chapter in this book corresponds to a different stage of a typical single-cell data analysis project. Generally, an analysis workflow would follow the order of the chapters with some flexibility concerning downstream analysis objectives. All of our chapters feature extensive lists of references, and we encourage readers to consult the primary sources for our statements. Our summaries cannot always capture the full reasoning for our recommendations, although we try to provide the required background whenever possible.
+Each chapter corresponds to a distinct phase of a typical single-cell data analysis project. While an analysis workflow should generally follow the order of the chapters, flexibility is encouraged depending on specific downstream analysis objectives. Each chapter is supplemented with extensive references, and readers are encouraged to consult these primary sources for a deeper understanding. Although we strive to provide comprehensive background information, our summaries and the [glossary](glossary.md) may not capture the full scope of reasoning behind every recommendation.
 
 ## Prerequisites
 
-Bioinformatics is a challenging research field for newcomers as it requires knowledge in both biology and computer science. Single-cell is even more demanding as it combines many subfields and datasets are often large. This book cannot cover all prerequisites for computational single-cell analysis, we therefore recommend a coarse overview of various topics below. The following links might increase your learning experience throughout the book:
+Bioinformatics is inherently multidisciplinary, requiring knowledge of both biology and computer science. Single-cell analysis is particularly demanding, as it integrates multiple subfields and often involves large datasets. While this book cannot cover all the necessary foundational knowledge for computational single-cell analysis, we recommend the following resources to enhance your learning experience:
```

Comment:
```suggestion
Bioinformatics is inherently multidisciplinary, requiring knowledge of both biology and computer science.
Single-cell analysis is particularly demanding because it requires deep knowledge of multiple subfields.
While this book cannot cover all prerequisites for computational single-cell analysis, we recommend the following resources to enhance your learning experience:
```